test/integration/test_backend.py

```diff
@@ -190,3 +191,22 @@ def test_too_many_qubits_in_circuit(self):
                 f"Circuit contains {num_qubits} qubits, but backend has only {num}.",
                 str(err.exception),
             )
+
+    def test_job_backend_properties(self):
+        """Test job backend properties."""
+        backends = self.dependencies.provider.backends(
+            instance=self.dependencies.instance, simulator=False
+        )
+        backend = least_busy(backends)
+        job = backend.run(ReferenceCircuits.bell())
```

Comment:
Is there a way to test this without running a job on a real backend? This could really slow down and timeout our integration tests

Reply:
I changed the test to use `self.backend` which makes more sense. I also changed the test above that was doing the same. However, `self.backend` is actually also a real device. I don't think that is so bad as long as we only call `backend.run()` and don't actually retrieve results.

Reply:
You don't really need to run a new job. The code can instead select a previously run job (e.g. using `service.jobs()`) and run one only if no previous jobs are found.